index.html

```diff
@@ -0,0 +1,222 @@
+            </li>
+            <li class="activity-description">
+              Lorem ipsum dolor sit amet consectetur adipisicing elit. Veniam corporis consequuntur libero odit accusantium amet impedit quo? Modi?
+            </li>
+          </ul>
+        </li>
+        <li>
+          <ul>
+            <li class="icon">
+              <i class="fa-solid fa-comments"></i>
+            </li>
+            <li class="activity">
+              Applications
+            </li>
+            <li class="activity-description">
+              Lorem ipsum dolor sit amet consectetur adipisicing elit. Tempora doloribus ratione, magnam voluptates rem eius!
+            </li>
+          </ul>
+        </li>
+        <li>
+          <ul>
+            <li class="icon">
+              <i class="fa-solid fa-code"></i>
+            </li>
+            <li class="activity">
+              Hackathon
+            </li>
+            <li class="activity-description">
+              Lorem ipsum dolor sit amet consectetur, adipisicing elit. Ipsa vitae deserunt, error dicta voluptas repellat dignissimos.
+            </li>
+          </ul>
+        </li>
+        <li>
+          <ul>
+            <li class="icon">
+              <i class="fa-brands fa-connectdevelop"></i>
+            </li>
+            <li class="activity">
+              Network
+            </li>
+            <li class="activity-description">
+              Lorem ipsum dolor sit amet consectetur adipisicing elit. Mollitia pariatur sed iusto sint praesentium aut quibusdam, suscipit cum asperiores magni.
+            </li>
+          </ul>
+        </li>
+      </ul>
+      <button id="join-summit">
+        Join Blockchain Summit 2023
+      </button>
+    </div>
+    <div id="speakers">
+      <h2 class="section-title">Featured Speakers</h2>
+      <hr class="section-heading-underline">
+      <ul id="featured-speakers">
+        <!-- added dynamically in javascript -->
+      </ul>
+      <button id="more-featured-speakers">
+        MORE
+        <i class="fa-solid fa-angle-down" id="more-featured-speakers-dropdown"></i>
+      </button>
+    </div>
+    <div class="partners home-hidden">
+      <h2>Partner</h2>
+      <hr class="section-heading-underline">
+      <!-- partner logos go here -->
+      <div id="partner-logos">
+        <img src="./images/partners/chainlink-link-logo.svg" alt="Chainlink">
+        <img src="./images/partners/cardano-ada-logo.svg" alt="Cardano">
+        <img src="./images/partners/solana-sol-logo.svg" alt="Solana">
+        <img src="./images/partners/Ethereum-Landscape-Black-Logo.wine.svg" alt="Ethereum">
+        <img src="./images/partners/google.svg" alt="Google">
+      </div>
+    </div>
+  </main>
+  <footer class="home-hidden">
+    <img src="./images/blockchain-logo.png" alt="Logo" class="footer-logo">
+    <div class="footer-description">
+      <p>2023 Blockchain On-ramp Summit.</p>
+      <p>Some rights reserved.</p>
+    </div>
+  </footer>
```

Comment:
- [ ] While you do a great effort 😃 I see that the padding of the footer is less than the Behance design, so please 😊 try to increase the padding vertically to be like the Behance 💯
- [ ] Also please 😃consider stick to the Behance design as you see here in the footer in the Behance is light mode and you made it dark ,so please try to make it like the Behance 

> Check below to see the difference 😊⤵️
> If you can not see the image clearly just click on it to open a new tab in the browser😊

| Figma design | Your design| 
| --- | --- |
|  ![image](https://user-images.githubusercontent.com/83360636/225302468-39366a37-a41c-4a87-ad42-52b9e7c54f12.png)  |    ![image](https://user-images.githubusercontent.com/83360636/225302416-abd76047-b254-41b3-aeab-98c46edeac5f.png)  |